mathics/doc/latex_doc.py

```diff
@@ -79,8 +79,8 @@
     r"(?P<content>.*?)\\end\{(?P=tag)\}"
 )
 
-LATEX_TESTOUT_DELIM_RE = re.compile(r",")
-NUMBER_RE = re.compile(r"(\d*(?<!\.)\.\d+|\d+\.(?!\.)\d*|\d+)")
+LATEX_TESTOUT_DELIM_RE = re.compile(r", ")
+NUMBER_RE = re.compile(r"([ -])(\d*(?<!\.)\.\d+|\d+\.(?!\.)\d*|\d+)")
```

Comment:
This deserves a small comment: without this change,
```
\includegraphics[width=5cm,height=3cm]{/tmp/img.png}
```
is converted into
```
\includegraphics[width=\allowbreak{}5\allowbreak{}cm,\allowbreak{}height=\allowbreak{}3\allowbreak{}cm]{/tmp/img.png}
```
because the parameters are treated as numbers to be displayed.

Here I exploded the fact that positive numbers in the parameters do not have spaces in front of them, different from the real output text. In any case, this conversion code should be removed at some point, in favor of a more standard way to parse LaTeX/markdown.

Reply:
Thanks for the information. If it deserves a comment in a PR commit, it deserves a comment in the code.  Thanks.

Reply:
Looks much better - thanks!